apps/ui/src/networks/offchain/constants.ts

```diff
@@ -5,7 +5,420 @@ export const SUPPORTED_EXECUTORS = {};
 export const RELAYER_AUTHENTICATORS = {};
 export const AUTHS = {};
 export const PROPOSAL_VALIDATIONS = {};
-export const STRATEGIES = {};
+export const STRATEGIES = Object.fromEntries(
+  [
+    'cap-voting-power',
```

Comment:
Why do we need to hardcode the list here?

Reply:
Other way will be to import the `strategies` repo

Reply:
Just to get a list? Could be with this endpoint: https://score.snapshot.org/api/strategies but why do you need the list in first place? Don't you get the strategies ids from the proposal or space already?

Reply:
This piece of code is pulling the strategy name from the static list https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/components/Modal/VotingPower.vue#L43

And later (for offchain voting), we have to use `SUPPORTED_STRATEGIES` here https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/networks/offchain/constants.ts#L3, which will use the static list again.

Reply:
Would be nice to find a workaround so we dont rely on some hardcoded list

Reply:
But how will that resolve the next case, the `SUPPORTED_STRATEGIES` constant ? Either way, we do need a list, unless we make an exception to bypass it for offchain votes

Reply:
Good point, let me take a look how to work it around.